examples/mp-openapi/src/test/java/org/superbiz/openapi/WeatherServiceTest.java

```diff
@@ -0,0 +1,203 @@
+                "                \"deprecated\": false,\r\n" + 
+                "                \"description\": \"Describes how the day will be.\",\r\n" + 
+                "                \"operationId\": \"dayStatus\",\r\n" + 
+                "                \"parameters\": [\r\n" + 
+                "                    {\r\n" + 
+                "                        \"allowEmptyValue\": false,\r\n" + 
+                "                        \"allowReserved\": false,\r\n" + 
+                "                        \"description\": \"The day for which the weather needs to be fetched.\",\r\n" + 
+                "                        \"in\": \"path\",\r\n" + 
+                "                        \"name\": \"\",\r\n" + 
+                "                        \"required\": true,\r\n" + 
+                "                        \"schema\": {\r\n" + 
+                "                            \"type\": \"string\"\r\n" + 
+                "                        }\r\n" + 
+                "                    }\r\n" + 
+                "                ],\r\n" + 
+                "                \"responses\": {\r\n" + 
+                "                    \"400\": {\r\n" + 
+                "                        \"content\": {\r\n" + 
+                "                            \"200\": {}\r\n" + 
+                "                        },\r\n" + 
+                "                        \"description\": \"Weather for this day not found\"\r\n" + 
+                "                    }\r\n" + 
+                "                },\r\n" + 
+                "                \"summary\": \"Finds weather for day specified in the URL \"\r\n" + 
+                "            }\r\n" + 
+                "        }";
+        JsonObject expectedJson = Json.createReader(new StringReader(expected)).readObject();
+        assertEquals(expectedJson.keySet().size(), response.keySet().size());
+        String[] expectedKeys = new String[]{"deprecated", "description", "operationId", "parameters", "responses", "summary"};
+        Stream.of(expectedKeys).forEach((text) -> {
+            assertTrue("Expected: " + text
+                    + " to be present in " + expected,
+                    expectedJson.getJsonObject("get").get(text) != null);
+        });
+
+    }
+
+    @Test
+    public void detailedDayStatus() {
+        WebTarget webTarget = this.client.target(this.base.toExternalForm());
+        final Response message = webTarget.path("/openapi")
+                .request()
+                .accept(MediaType.APPLICATION_JSON)
+                .get();
+        final String metaData = message.readEntity(String.class);
+        JsonObject jsonObject = Json.createReader(new StringReader(metaData)).readObject();
+        JsonObject response = jsonObject.get("paths").asJsonObject().get("/weather/detailedWeather/{day}").asJsonObject();
+        final String expected = "{\r\n" + 

```

Comment:
This json doesn't look well too, but this is an issue that needs to be reported as well. Like I said before there are properties that are no in the Weather class.

Reply:
I'll raise an issue regarding this.